src/crimson/os/alienstore/alien_store.cc

```diff
@@ -180,10 +181,9 @@ seastar::future<> AlienStore::umount()
     // not really started yet
     return seastar::now();
   }
-  return op_gate.close().then([this] {
-    return tp->submit([this] {
+  op_gates[seastar::this_shard_id()].close().get();
```

Comment:
It looks not meaningful to only close a particular sharded gate. During a shutdown, I think to make sure AlienStore is inactive, the gate in all shards need to be closed.

Reply:
will use the close_all